tests/unit/request/node.ts

```diff
@@ -122,6 +122,12 @@ const responseData: { [url: string]: DummyResponse } = {
 	}
 };
 
+process.on('unhandledRejection', (reason: any, promise: any) => {
```

Comment:
Is this still desired/intended?  Seems like it was what was used to chase down the issues.